CanHazFunny/CanHazFunny/PrintToConsoleService.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+
+namespace CanHazFunny
+{
+    public class PrintToConsoleService : IPrint
+    {
+
+        public void Print(string text)
+        {
+            if (text == null) throw new ArgumentNullException(nameof(text));
+            if (text.Length == 0) throw new ArgumentException(nameof(text));
```

Comment:
```suggestion
            if(string.IsNullOrWhiteSpace(text)) throw new ArgumentException($"{nameof(text)} should not be null or empty");
```
You can use `IsNullOrWhiteSpace` or `IsNullOrEmpty` to do this in one line. It is also a good idea to give a better description with your `ArgumentException`.